<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto Writing</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit1"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edit3"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Access Token</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Blog Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Problem</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edit2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ex_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tistory BOJ auto writing</string>
   </property>
  </widget>
  <widget class="QLabel" name="developmenter">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>made by 삼성유망주</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Result View</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="result_scroll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>382</width>
      <height>149</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>label_4</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>ex_name</zorder>
  <zorder>developmenter</zorder>
  <zorder>btn_enter</zorder>
  <zorder>btn_reset</zorder>
  <zorder>result_label</zorder>
  <zorder>btn_save</zorder>
  <zorder>btn_load</zorder>
  <zorder>result_scroll</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
